lib/pry/repl.rb

```diff
@@ -79,6 +82,8 @@ def repl
           return pry.exit_value unless pry.eval(val)
         end
       end
+    ensure
+      trap('SIGINT', original_interrupt_handling)
```

Comment:
We cannot rely on ensure https://www.schneems.com/2017/02/21/the-oldest-bug-in-ruby-why-racktimeout-might-hose-your-server/

We may want to include an at_exit hook or rescue SystemExit